### DISCUSSION TITLE
local cross-seed --> remote rtorrent

### DISCUSSION THREAD

**Discussion by rj-d2** [cross-seed/cross-seed#275] [General]
Hi,

im still running version 2.9.1 on my seedbox, i cannot run v3/4 because of this issue --> https://github.com/mmgoodnow/cross-seed/issues/178

atm im trying to figure out if it would make sense to run cross-seed locally on my nas with my rtorrent beeing on a seedbox. 🤔️

plan:
- sync/mount rtorrent_session directory to my nas
- start cross-seed docker, let it search and inject to rtorrent

problems:
atm im using rsync to sync the rtorrent session folder to my nas,
im seeding ~1400 torrents atm, there are about 4100 files in the rtorrent session folder, 3 items per torrent --> .torrent, .rtorrent, .libtorrent_resume
do i need to sync all of them for cross-seed to work?
im not sure if this workflow is a good idea anyway, im kinda worried about my ssd getting hammered 🔨️, 
would mounting the session folder to my nas be a better idea?

cross-seed is searching and finding torrents, but cannot inject, it saves the torrent files instead.
atm im using rsync to move the torrents found by cross-seed to my seedbox and use autotorrent on the seedbox to add the torrrents
can cross-seed inject to a remote rtorrent or does it needs to be the same host?

result:
running v4.0.1 the first time added ~450 new torrents compared to cross-seed v2.9.1 running on my seedbox. 😍️
needless to say you are doing a great job 👍️

is anyone here doing something similar?
any idea to make this work better?

**Comment by mmgoodnow** [comment]
Seems like you figured out a solution that works well for you. Yes, I think backing up the session directory to your NAS is a good idea here. 

> do i need to sync all of them for cross-seed to work?

Yes you do. However they don't change much so I wouldn't be too worried about hammering your SSD.

**Comment by mmgoodnow** [comment]
> can cross-seed inject to a remote rtorrent or does it needs to be the same host?

It needs to be the same host. `cross-seed` basically hits rTorrent's API with "here's the path to a torrent file stored locally, and also we already have the data. load it in and start it"

The only thing I could think of is if you had the outputDir as a network share.
